content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -68,6 +68,22 @@ or contain bugs.
 
 Skipped CSVs do not need to be present in a catalog or set of manifests prior to adding to a catalog.
 
+#### Example for replaces and skips
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: etcdoperator.v0.9.2
+  namespace: placeholder
+spec:
+    displayName: etcd
+    description: Etcd Operator
+    replaces: etcdoperator.v0.9.0
+    skips:
+    - etcdoperator.v0.9.1
+```
+
```

Comment:
Same here. Do we need this example? Have we not an example already?

Reply:
We decided to add multiple examples that's why added few more examples.

Reply:
What's the motivation behind multiple examples? Are they communicating something different in every example? Unless we have so salient difference we're trying to communicate, more example will just end up confusing the reader (remember that more often than not, folks don't read documentation patiently :) they just want the info they need and get out of there)